training/model_management/environments/model-management/context/requirements.txt

```diff
@@ -1,7 +1,11 @@
 azure-core=={{latest-pypi-version}}
 azure-ai-ml=={{latest-pypi-version}}
 azure-identity=={{latest-pypi-version}}
+azureml-contrib-services=={{latest-pypi-version}}
```

Comment:
what's in contrib services, I thought that was deprecated long time ago

Reply:
still being released: https://pypi.org/project/azureml-contrib-services/ here. Let me know if I need to remove this. 
Its used in the scoring script only

Reply:
Removed. @vizhur please help to check

Reply:
Let us get first set of models to Prod and then handle the next set that require these new packages.